content/en/docs/languages/php/context.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Context
+weight: 55
+cSpell:ignore: Swoole
+---
+
+In order for OpenTelemetry to work, it must store and propagate important
+telemetry data. For example, when a request is received and a span is started it
+must be available to a component which creates its child span. To solve this
+problem, OpenTelemetry stores the span in the Context.
+
+More information:
+
+- [Context specification](/docs/specs/otel/context/)
+
+## Context
+
+Context is globally available, and there can only be one
+[active context](#active-context) in the current execution context. Context can
+store values (for example, a `Span`).
+
+### Storage
+
+Context uses `Storage` to keep track of values. By default, a generic
+`ContextStorage` is used. OpenTelemetry for PHP supports other context storage
+for more exotic use-cases (eg for asynchronous/concurrent execution with
+`fibers`).
+
+## Context Keys
+
+Context entries are key-value pairs. Keys can be created by calling
+`OpenTelemetry\Context\Context::createKey()`.
+
+```php
+use OpenTelemetry\Context\Context;
+
+$key1 = Context::createKey('My first key');
+$key2 = Context::createKey('My second key');
+```
+
+Context keys are used to store and retrieve values from context.
+
+## Basic Operations
+
+### Store and Retrieve values
+
+Values are stored in Context by using the `$context->with($key, $value)` method.
+Setting a context entry creates a new context with the new entry in its storage,
+containing `$value`.
+
+Context is immutable, and setting a context entry does not modify the previous
+context. Once a value is stored in a context, it can be retrieved via
+`$context->get($key)`:
```

Comment:
```suggestion
Values are stored in context by using the `$context->with($key, $value)` method.
Setting a context entry creates a new context with the new entry in its storage,
containing `$value`.

Context is immutable, and setting a context entry does not modify the previous
context. Once a value is stored in a context, it can be retrieved using
`$context->get($key)`:
```

@svrnm Not sure if we're to follow the K8s way here and capitalize entities from the specs, like Context. Was this ever discussed? Wouldn't want to miss on something here.

Reply:
I did capitalize it because it's a key OTEL term...

Reply:
let's keep it capitalized, @theletterf we might add a list of that in the style guide